core/objects.go

```diff
@@ -228,30 +228,34 @@ func (r *Registration) MergeUpdate(input Registration) {
 // challenge, we just throw all the elements into one bucket,
 // together with the common metadata elements.
 type Challenge struct {
+	ID int `json:"id,omitempty" db:"id"`
+
+	AuthorizationID string `json:"authorizationID,omitempty" db:"authorizationID"`
+
 	// The type of challenge
-	Type string `json:"type"`
+	Type string `json:"type" db:"type"`
```

Comment:
Let's not continue this trend. We keep ending up having to separate these into db models vs json view structs anyway.

Plus, this doesn't have the usual LockCol on it. Is it needed? Do we never allow updates to this?